src/App.vue

```diff
@@ -1,21 +1,65 @@
 <template>
-  <div id="GamePanel"><GamePanel /></div>
+  <div id="app">
+    <div id="GamePanel"><GamePanel /></div>
+    <ContentModal
```

Comment:
Why is the ConentModal placed outside of the GamePanel?

Reply:
When it was nested inside multiple components (inside the MemoryCard) it was impossible to let it cover the entire screen. There was an issue caused by the SummaryPanel. To cover the entire screen it either needed the App as a direct parent (as it is now) or it needs to be inside the GamePanel and some css to make it larger than the GamePanel(negative margin). So I thought it was more logical to directly put it in the app instead of putting it in the GamePanel only to then revert the size changes made in the GamePanel.

Reply:
Oh I see the problem is the padding of the GamePanel. Then it would probably make sense to create an extra container inside GamePanel that is the flexbox and has padding to create the border at the edges. After that, you can put the Modal inside the GamePanel on the same level as the container. In my opinion that would make more sense because the ContentModal is logically part of the GamePanel and will only be used there, too if I'm not wrong.